index.json

```diff
@@ -53,13 +53,22 @@
   "globalid": {
     "dependencies": [
       "activerecord"
-     ],
+    ],
```

Comment:
This snuck in from https://github.com/Shopify/rbi-central/pull/134 and it seems CI only checks the index when the file has changed and not all the time, therefore main is green ([see eg this merge](https://github.com/Shopify/rbi-central/actions/runs/3696918484/jobs/6261262917))